cypress/support/stackable/commands/toolbar.js

```diff
@@ -105,4 +105,8 @@ export function adjustDynamicContent( blockName, blockSelector, selector, option
 			.find( 'button.apply-changes-button' )
 			.click( { force: true } )
 	} )
+
+	cy.savePost()
+	// Sometimes the first save does not register and the test fails.
+	cy.savePost()
```

Comment:
why is it failing?

Reply:
cannot reproduce the error anymore when there's only one `cy.savePost()`. Will remove the extra save.